src/NumericInput.js

```diff
@@ -49,6 +49,12 @@ export default class NumericInput extends PureComponent {
         this.props.onChange(e);
     }
 
+    componentWillUpdate(nProps, nState){
+        if (this.state.value.length - nState.value.length === 2) {
```

Comment:
What if user backspace so fast that

`nState.value = 1` with `this.state.value = 101` ?

This would make length diff == 2, am I correct?